stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -66,7 +66,7 @@ def assign_quants(self):
         if move.picking_type_id.auto_fill_qty_done:
             # Auto-fill all lines as done
             for ml in move.move_line_ids:
-                ml.qty_done = ml.reserved_uom_qty
+                ml.quantity = ml.quantity
```

Comment:
I need help for this part. What functionality does "auto_fill_qty_done" need to have? In V17 there is no distinction between reserved and done quantity on stock move lines as far as I can see.